docs/docs/about/references/keywords/RATE.md

```diff
@@ -44,20 +44,27 @@ RATE: SIM1:GAS_PROD
 ~~~~~~~~
 
 ## Use in EMISSION for VENTING_EMITTERS (from eCalc v8.8)
-The attribute `VALUE` is required, while `UNIT` and `TYPE` are optional. Allowed values for
-`UNIT` are kg/d and t/d, while STREAM_DAY and CALENDAR_DAY are valid for `TYPE`.
+The keywords `VALUE` and [CATEGORY](/about/references/keywords/CATEGORY.md) are required, while [UNIT](/about/references/keywords/UNIT.md) and [TYPE](/about/references/keywords/TYPE.md) are optional. 
```

Comment:
same here?

Reply:
Same here, has been like this since the TYPE was introduced (8.13, described under VENTING_EMITTERS), see comment above.

Reply:
it also helps us remember when looking at the documentation, instead of having to look at changelog